pages/trouw-opties/index.tsx

```diff
@@ -67,29 +65,25 @@ export default function MultistepForm1() {
                   {/*TODO: Step indicator component */}
                   <Paragraph lead>{t("common:step-n-of-m", { n: 1, m: 5 })} — Kies wat je wil</Paragraph>
                 </HeadingGroup>
-                <form onSubmit={onWeddingOptionsSubmit}>
-                  <Heading2>Wij willen trouwen</Heading2>
-                  <Button
-                    type="submit"
-                    name="type"
-                    value="huwelijk"
-                    appearance="primary-action-button"
-                    onClick={onWeddingOptionsClick}
-                  >
-                    Trouwen plannen <UtrechtIconArrow />
-                  </Button>
-                  <Heading2>Wij willen een geregistreerd partnerschap</Heading2>
-                  <Button
-                    type="submit"
-                    name="type"
-                    value="geregistreerd-partnerschap"
-                    appearance="primary-action-button"
-                    onClick={onWeddingOptionsClick}
-                  >
-                    Geregistreerd partnerschap plannen
-                    <UtrechtIconArrow />
-                  </Button>
-                </form>
+                <Heading2>Wij willen trouwen</Heading2>
```

Comment:
Refactored this because the form submit version with just two buttons didn't make a lot of sense to me. It relied on the button onClick running it's state update _before_ the form submit is processed. Unless there is a good reason from a11y perspective to use a form here, this seems like a better approach.